main/tests.py

```diff
@@ -76,4 +76,25 @@ def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_ow
         client.force_login(user=create_dog_owner_user.user)
         response = client.get("/homepage/")
         assert response.status_code == 200
-        assert list(response.context['daycares']) == list(DayCare.objects.all())
+        list_of_daycares_from_response = list(response.context['daycares'])
+        list_of_all_daycares = list(DayCare.objects.all())
+        assert list_of_daycares_from_response == list_of_all_daycares
+
+
+@pytest.mark.django_db
+class TestNavbarView:
+    def test_navbar_getting_dog_owner_nickname_and_profile_picture(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/homepage/")
+        dog_owner_picture_url = create_dog_owner_user.dog_picture_url
+        assert response.context['navbar_picture_url'] == dog_owner_picture_url
```

Comment:
I would pass variables substitutions in cases like this, and assert the actual value directly instead:
```
assert response.context['navbar_picture_url'] ==  create_dog_owner_user.dog_picture_url
```
Also, it would make the test look nicer if the `assert` line would be separated with new line from the test setup logic.